rs/rosetta-api/tests/system_tests/common/constants.rs

```diff
@@ -0,0 +1,2 @@
+pub const STARTING_CYCLES_PER_CANISTER: u128 = 2_000_000_000_000_000;
+pub const DEFAULT_INITIAL_BALANCE: u64 = 1_000_000_000;
```

Comment:
Would it make sense to have these constants, and potentially everything else that you are now adding under `rs/rosetta-api/tests/system_tests/common`, in one or more separate `test_utils` crate(s)?

Reply:
There already exists a test_utils crate. It contains a very old and complex version of an icp rosetta client. The content in `rs/rosetta-api/tests/system_tests/common` are only meant to be used by the tests inside of `rs/rosetta-api/tests/system_tests` not anywhere else in the repo.

Reply:
I could imagine having more than just a single `test_utils` crate, e.g., one generic one for Rosetta, and individual ones for the ICP and ICRC versions, etc. I think this is better discussed in the context of the larger reorg/refactoring epic, so having this as you propose here is fine!